tpc/pregen/media.dsl/dslplatform/Guards.java

```diff
@@ -0,0 +1,15 @@
+package serializers.dslplatform;
+
+public final class Guards {
+    /// No-oped for perf reasons
```

Comment:
I understand.  In that case it seems best to revert it.

(Even though it's not part of serialization, it is included in some of the measurements.)